setup.py

```diff
@@ -42,7 +42,8 @@
     'edtf>=4.0.1,<5.0.0',
     'ftfy>=4.4.3',
     'geojson>=2.5.0',
-    'marshmallow>=3.0.0,<4.0.0',
+    'idutils>=1.1.8',
+    'marshmallow>=3.5.0,<4.0.0',
```

Comment:
Need to bump so `Nested` can get a `partial`, before it only supported `Schema`.